scripts/src/test/bash/integration-test-java-extra

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cd ../test-setup
+CLI="${PWD}/scripts/devon"
+EXTRA_VERSION=$(head -n 1 "${PWD}"/mirrors/java/available-versions)
```

Comment:
this would be the same as `JAVA_VERSION` that is undefined and would also lead to the same latest version.
In that case, can you also set `JAVA_VERSION` to some 11x version - why not `11*` to test the new feature of version prefix?